src/lib/sql/sql_translator.cpp

```diff
@@ -1740,13 +1742,13 @@ std::shared_ptr<AbstractExpression> SQLTranslator::_translate_hsql_expr(
                       "Interval can only be applied to ValueExpression with String value");
           const auto start_date_string =
               std::string{boost::get<pmr_string>(static_cast<ValueExpression&>(*left).value)};
-          const auto start_date = string_to_date(start_date_string);
+          const auto start_date = string_to_timestamp(start_date_string);
```

Comment:
start_timestamp?

Reply:
or add `->date()` since we don't use the timestamp anyway?

Reply:
I'll go for the first suggestion. The second one will segfault if the given date is invalid.